packages/core/src/routes/sign-in-experience/index.openapi.json

```diff
@@ -132,6 +132,36 @@
           }
         }
       }
+    },
+    "/api/sign-in-exp/default/check-password": {
```

Comment:
Need a custom operation name. The openapi endpoint integration tests failed. Can't generate the operation name automatically based on the path.